mage_ai/data_preparation/models/block/__init__.py

```diff
@@ -258,6 +259,7 @@ def __init__(
         self.color = block_color
         self.configuration = configuration
         self.has_callback = has_callback
+        self.timeout = timeout
```

Comment:
Do you need to update the BlockPolicy?

Reply:
good call, i forgetting to turn on user auth since it always gets reset